openprescribing/frontend/management/commands/measure_definitions/bdzadq.json

```diff
@@ -12,7 +12,8 @@
     "anxiety and sleep problems. <a href='https://www.rcpsych.ac.uk/expertadvice/treatments/benzodiazepines.aspx'> The Royal College of Psychiatrists states</a> ",
     "that \"around 4 in every 10 people who take them every day for more than 6 weeks will become addicted\" ",
     "and therefore they should not be prescribed for longer than 4 weeks.  This measure shows the mean Average Daily Quantity (ADQ) given per prescription, ",
-    "for both the older benzodiazepines, such as diazepam and temazepam, and the newer \"Z-drugs\", such as zopiclone."
+    "for both the older benzodiazepines, such as diazepam and temazepam, and the newer \"Z-drugs\", such as zopiclone.  It excludes items that do not ",
```

Comment:
Should this be "It excludes items that do not **have**..."?

Reply:
yes, have amended.